doc/start/quick-cephfs.rst

```diff
@@ -3,117 +3,210 @@
-   clean``  state. Also, ensure that you have at least one :term:`Ceph Metadata
-   Server` running. :: 
+   clean`` state. ::
 
 	ceph -s [-m {monitor-ip-address}] [-k {path/to/ceph.client.admin.keyring}]
 
 
-Create a Filesystem
-===================
+Deploy Metadata Server
+======================
+
+All metadata operations in CephFS happen via a metadata server, so you need at
+least one metadata server. Execute the following to create a metadata server::
+
+  ceph-deploy mds create {ceph-node}
+
+For example::
+
+  ceph-deploy mds create node1
+
+Now, your Ceph cluster would look like this:
+
+.. ditaa::
+           /------------------\         /----------------\
+           |    ceph-deploy   |         |     node1      |
+           |    Admin Node    |         | cCCC           |
+           |                  +-------->+   mon.node1    |
+           |                  |         |     osd.0      |
+           |                  |         |   mgr.node1    |
+           |                  |         |   mds.node1    |
+           \---------+--------/         \----------------/
+                     |
+                     |                  /----------------\
+                     |                  |     node2      |
+                     |                  | cCCC           |
+                     +----------------->+                |
+                     |                  |     osd.1      |
+                     |                  |   mon.node2    |
+                     |                  \----------------/
+                     |
+                     |                  /----------------\
+                     |                  |     node3      |
+                     |                  | cCCC           |
+                     +----------------->+                |
+                                        |     osd.2      |
+                                        |   mon.node3    |
+                                        \----------------/
+
+Create a File System
+====================
 
 You have already created an MDS (`Storage Cluster Quick Start`_) but it will not
-become active until you create some pools and a file system.  See :doc:`/cephfs/createfs`.
+become active until you create some pools and a file system.  See
+:doc:`/cephfs/createfs`. ::
 
-::
+    ceph osd pool create cephfs_data 32
+    ceph osd pool create cephfs_meta 32
+    ceph fs new mycephfs cephfs_meta cephfs_data
 
-    ceph osd pool create cephfs_data
-    ceph osd pool create cephfs_metadata
-    ceph fs new <fs_name> cephfs_metadata cephfs_data
+.. note:: In case you have multiple Ceph applications and/or have multiple
+   CephFSs on the same cluster, it would be easier to name your pools as
+   <application>.<fs-name>.<pool-name>. In that case, the above pools would
+   be named as cephfs.mycehfs.data and cephfs.mycehfs.meta.
 
+Quick word about Pools and PGs
+------------------------------
```

Comment:
This heading seems out of place in the rendered doc. Maybe it should have an '===' line under it instead?

Reply:
Not sure. I see this sub-section of doc as steps that reader might optionally following after/before/during the pool creation which the entire section is about. So I am OK with that.